src/Libraries/CoreNodeModels/Input/File.cs

```diff
@@ -321,8 +321,8 @@ public override void Dispose()
         }
     }
 
-    [NodeName("Directory.FromPath")]
-    [NodeCategory(BuiltinNodeCategories.CORE_IO)]
+    [NodeName("DirectoryFromPath")]
+    [NodeCategory("Core.File")]
```

Comment:
How would we migrate these nodes? Have you tried adding the new name to the `AlsoKnownAs` category?